api/build.gradle

```diff
@@ -1,65 +1 @@
-            [group: 'org.hibernate', name: 'hibernate-validator', version: versions.hibernatevalidator],
-            [group: 'com.google.code.gson', name: 'gson'],
-            [group: 'org.apache.fineract.cn', name: 'api', version: versions.frameworkapi]
-    )
-
-    testCompile(
-            [group: 'org.apache.fineract.cn', name: 'test', version: versions.frameworktest],
-    )
-}
-
-publishing {
-    publications {
-        apiPublication(MavenPublication) {
-            from components.java
-            groupId project.group
-            artifactId project.name
-            version project.findProperty('externalVersion') ?: project.version
-        }
-    }
-}
+/*
 * Licensed to the Apache Software Foundation (ASF) under one or more
 * contributor license agreements.  See the NOTICE file distributed with
 * this work for additional information regarding copyright ownership.
 * The ASF licenses this file to You under the Apache License, Version 2.0
 * (the "License"); you may not use this file except in compliance with
 * the License.  You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

buildscript {

    repositories {
        jcenter()
    }

    dependencies {
        classpath 'io.spring.gradle:dependency-management-plugin:1.0.9.RELEASE'
    }
}

plugins {
    id 'com.github.hierynomus.license' version '0.15.0'
    id("org.nosphere.apache.rat") version "0.6.0"
    id "com.jfrog.artifactory" version "4.15.1"
    id "io.spring.dependency-management" version "1.0.9.RELEASE"
    id 'maven-publish'
    id 'java'
    id "com.github.ben-manes.versions" version "0.28.0"

}


apply from: '../shared.gradle'

apply plugin: 'io.spring.dependency-management'

dependencies {
    compile (
            [group: 'org.springframework.cloud', name: 'spring-cloud-starter-openfeign'],
            [group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'],
            [group: 'org.hibernate', name: 'hibernate-validator', version: versions.hibernatevalidator],
            [group: 'com.google.code.gson', name: 'gson'],
            [group: 'org.apache.fineract.cn', name: 'api', version: versions.frameworkapi]
    )

    testCompile(
            [group: 'org.apache.fineract.cn', name: 'test', version: versions.frameworktest],
    )
}

publishing {
    publications {
        apiPublication(MavenPublication) {
            from components.java
            groupId project.group
            artifactId project.name
            version project.findProperty('externalVersion') ?: project.version
        }
    }
}

```

Comment:
Something is wrong with newline characters. please check.

Reply:
Yes might fix this in the next PR